terraform/shared/shared.tf.json.template.py

```diff
@@ -181,7 +194,14 @@
                 's3_bucket_name': '${aws_s3_bucket.shared_cloudtrail.id}',
                 'enable_log_file_validation': True,
                 'is_multi_region_trail': True,
```

Comment:
Consider adding …
```suggestion
                'is_multi_region_trail': True,
                'enable_logging': False,
```
… [tf cloudtrail enable_logging](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudtrail#enable_logging).